torch/lib/c10d/ProcessGroupNCCL.hpp

```diff
@@ -237,12 +246,9 @@ class ProcessGroupNCCL : public ProcessGroup {
     // callback's stream and will have the result value of the callback.
     void addCallbackWithStream(
         std::function<void(void)> callback,
-        const c10::cuda::CUDAStream& stream,
-        std::shared_ptr<std::vector<at::cuda::CUDAEvent>> thenFutCudaEvents) {
+        const c10::cuda::CUDAStream& stream) {
```

Comment:
We don't need to explicitly pass the stream in here right? We can just use `futureNCCLCallbackStream_` directly in the method?